examples/DynaCLR/vcp_tutorials/quickstart.ipynb

```diff
@@ -0,0 +1,731 @@
+    "annotation = pd.read_csv(annotations_path)\n",
+    "\n",
+    "# Strip whitespace from fov_name to match features\n",
+    "annotation[\"fov_name\"] = annotation[\"fov_name\"].str.strip()\n",
+    "\n",
+    "# Merge on (fov_name, track_id, t) as these uniquely identify each cell observation\n",
+    "annotation_indexed = annotation.set_index([\"fov_name\", \"track_id\", \"t\"])\n",
+    "mi = pd.MultiIndex.from_arrays(\n",
+    "    [\n",
+    "        features_anndata.obs[\"fov_name\"],\n",
+    "        features_anndata.obs[\"track_id\"],\n",
+    "        features_anndata.obs[\"t\"],\n",
+    "    ],\n",
+    "    names=[\"fov_name\", \"track_id\", \"t\"],\n",
+    ")\n",
+    "features_anndata.obs[\"annotations_infections_state\"] = annotation_indexed.reindex(mi)[\n",
+    "    ANNOTATION_COLUMN\n",
+    "].values\n",
+    "\n",
+    "# Plot the PCA and PHATE embeddings colored by infection state\n",
+    "# Prepare data for plotting\n",
+    "# Map numeric labels to readable labels for legend\n",
+    "infection_state_labels = {0: \"Unknown\", 1: \"Uninfected\", 2: \"Infected\"}\n",
+    "\n",
+    "plot_df = pd.DataFrame(\n",
+    "    {\n",
+    "        \"PC1\": features_anndata.obsm[\"X_pca\"][:, 0],\n",
+    "        \"PC2\": features_anndata.obsm[\"X_pca\"][:, 1],\n",
+    "        \"PHATE1\": features_anndata.obsm[\"X_phate\"][:, 0],\n",
+    "        \"PHATE2\": features_anndata.obsm[\"X_phate\"][:, 1],\n",
+    "        \"infection_state\": features_anndata.obs[\"annotations_infections_state\"]\n",
+    "        .fillna(0)\n",
+    "        .map(infection_state_labels),\n",
+    "    }\n",
+    ")\n",
+    "\n",
+    "# Define color palette (colorblind-friendly: blue for uninfected, orange for infected)\n",
+    "color_palette = {\n",
+    "    \"Unknown\": \"lightgray\",  # Unlabeled\n",
+    "    \"Uninfected\": \"cornflowerblue\",  # Uninfected\n",
+    "    \"Infected\": \"darkorange\",  # Infected\n",
+    "}\n",
+    "\n",
+    "# Create figure with two subplots\n",
+    "fig, axes = plt.subplots(1, 2, figsize=(14, 6))\n",
+    "\n",
+    "# Plot PCA\n",
+    "sns.scatterplot(\n",
+    "    data=plot_df,\n",
+    "    x=\"PC1\",\n",
+    "    y=\"PC2\",\n",
+    "    hue=\"infection_state\",\n",
+    "    palette=color_palette,\n",
+    "    ax=axes[0],\n",
+    "    alpha=0.6,\n",
+    "    s=20,\n",
+    ")\n",
+    "axes[0].set_title(\"PCA Embedding\")\n",
+    "axes[0].set_xlabel(\"PC1\")\n",
+    "axes[0].set_ylabel(\"PC2\")\n",
+    "\n",
+    "# Plot PHATE\n",
+    "sns.scatterplot(\n",
+    "    data=plot_df,\n",
+    "    x=\"PHATE1\",\n",
+    "    y=\"PHATE2\",\n",
+    "    hue=\"infection_state\",\n",
+    "    palette=color_palette,\n",
+    "    ax=axes[1],\n",
+    "    alpha=0.6,\n",
+    "    s=20,\n",
+    ")\n",
+    "axes[1].set_title(\"PHATE Embedding\")\n",
+    "axes[1].set_xlabel(\"PHATE 1\")\n",
+    "axes[1].set_ylabel(\"PHATE 2\")\n",
+    "\n",
+    "plt.tight_layout()\n",
+    "plt.show()"
```

Comment:
This cell does not work for me, due to an issue with not finding the `extracted_inf_state.csv` file.

```
FileNotFoundError: [Errno 2] No such file or directory: 'extracted_inf_state.csv'
```

Did we fail to download this file in an earlier step?

Reply:
I figured out the bug. 1) I forgot to write the `wget` for downloading 2) the permission for the file were set to 750. Thanks!

Reply:
[ab92b74](https://github.com/mehta-lab/VisCy/pull/343/commits/ab92b74a51ec52dd6fc54e205795ea0c6f8cf895) the notebook now reflects the download of this data and the csv file needed to run the rest of the exercise